app/services/summarizers/main_summarizer.rb

```diff
@@ -1,23 +1,20 @@
 module Summarizers
   class MainSummarizer
     class << self
-      Results = Data.define(:eligibilities, :assessment_result)
-
-      def call(proceeding_types:, receives_qualifying_benefit:, receives_asylum_support:,
-               gross_income_eligibilities:, disposable_income_eligibilities:, capital_eligibilities:)
-        eligibilities = proceeding_types.map do |ptc|
+      def assessment_results(proceeding_types:, receives_qualifying_benefit:, receives_asylum_support:,
+                             gross_income_assessment_results:, disposable_income_results:, capital_assessment_results:)
+        values = proceeding_types.map do |ptc|
```

Comment:
@starswan Lets rename
- **ptc** to **pt**
- **disposable_income_result** to **disposable_income_assessment_result**